api/anilist/build.gradle.kts

```diff
@@ -33,15 +33,6 @@ android {
     }
 
     namespace = "com.imashnake.animite.api.anilist"
-
-    // Workaround for KSP generated sources not being indexable by the IDE
-    libraryVariants.all {
-        kotlin.sourceSets {
-            getByName(name) {
-                kotlin.srcDir("build/generated/ksp/$name/kotlin")
-            }
-        }
-    }
```

Comment:
This is no longer needed: https://kotlinlang.org/docs/ksp-quickstart.html#make-ide-aware-of-generated-code.